src/js/libcs/Operators.js

```diff
@@ -4530,3 +4530,68 @@ evaluator.dropped$0 = function() {
 evaluator.droppoint$0 = function() {
     return dropPoint;
 };
+
+evaluator.parsecsv$1 = function(args, modifs) {
+    // more strict float parsing
+    var filterFloat = function(value) {
+        if (/^(\-|\+)?([0-9]+(\.[0-9]+)?|Infinity)$/
+            .test(value))
+            return Number(value);
+        return NaN;
+    };
+
+    // autoconvert like tokenize
+    var aucon = true;
+    var mcon = evaluateAndVal(modifs.autoconvert);
+    if (mcon.ctype === "boolean") aucon = mcon.value;
+
+    var delim = "";
+    var md = modifs.delimiter;
+    if (md && md.ctype === "string" && md.value.length === 1) delim = md.value;
+
+    var nl = "";
+    var mnl = modifs.newline;
+    if (mnl && mnl.ctype === "string" && mnl.value.length === 2) nl = mnl.value;
```

Comment:
Why this length? I assume that the intention is to allow CRLF, but do we really benefit from disallowing a bare CR (Mac OS 9 style)? If the only valid value (other than the default) is CRLF, I'd prefer `mnl.value === "\r\n"` to make that explicit.